README.md

```diff
@@ -1,28 +1,27 @@
-# Bazel rules for container
+# Bazel rules for OCI containers
+
+**EXPERIMENTAL** This ruleset is highly experimental and not yet fit for production use.
+We might abandon it at any time, there is no guarantee of support or stability.
+
+This is a speculative alternative to rules_docker.
+
+We start from first principles and plan to avoid some pitfalls learned in maintaining that repo:
+
+- Use a toolchain consisting of off-the-shelf, pre-built layer and container manipulation tools (primarily [crane]).
+- Don't write language-specific rules, as we cannot be experts on all languages, nor can users deal with the versioning issues
+  that come with dependencies we might take on the rules for those languages.
+- Don't be docker-specific, now that it has a commercial license and other container runtimes exist.
+- Use our toolchain hermetically: don't assume there is a docker pre-installed on the machine.
+- Keep a tight complexity budget for the project so we are able to commit to effective maintenance.
+
+[crane]: https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane.md
```

Comment:
I am not sure about crane yet. the combination of skopeo and umoci falls stronger than crane. Shall we remove this statement for now? 

Also see: https://github.com/aspect-build/rules_container/issues/9

Reply:
sure I'll remove the specific